src/hooks/useIpc.js

```diff
@@ -1,23 +1,28 @@
 import React, { useEffect } from 'react';
-import { toast } from 'react-toastify';
-import { useDispatch } from 'react-redux';
+// import { toast } from 'react-toastify';
+// import { useDispatch } from 'react-redux';
 
-import htmlStringToReact from '@utils/htmlStringToReact';
-import { regex } from '@constants';
-import { addSearchParamsToUrl } from '@utils/searchParams';
-import { appUpdateAvailable } from '@actions';
+// import htmlStringToReact from '@utils/htmlStringToReact';
+// import { regex } from '@constants';
+// import { addSearchParamsToUrl } from '@utils/searchParams';
+// import { appUpdateAvailable } from '@actions';
 import FlashMessageHolder from '@toolbox/flashMessage/holder';
 import NewReleaseMessage from '@shared/newReleaseMessage/newReleaseMessage';
 
-const useIpc = (history) => {
-  const dispatch = useDispatch();
+const useIpc = () => {
+// const useIpc = (history) => {
+  // const dispatch = useDispatch();
 
-  const { ipc } = window;
+  // const { ipc } = window;
```

Comment:
why are we removing the IPC?

Reply:
In order to display the message. It will reverted once changes are approved.